ui/apps/platform/cypress/integration/general.test.js

```diff
@@ -35,22 +33,11 @@ describe('General sanity checks', () => {
         });
 
         it('for Violations', () => {
-            cy.intercept('GET', api.alerts.alerts).as('alerts');
-            cy.intercept('GET', api.alerts.alertscount).as('alertsCount');
-            cy.visit(violationsUrl);
-            cy.wait(['@alerts', '@alertsCount']);
+            visitViolations();
 
             cy.title().should('match', new RegExp(`Violations | ${productNameRegExp}`));
         });
 
-        it('for Violations with side panel open', () => {
-            cy.intercept('GET', api.alerts.alertById).as('alertById');
-            cy.visit('/main/violations/1234');
-            cy.wait('@alertById'); // 404
-
-            cy.title().should('match', new RegExp(`Violations | ${productNameRegExp}`)); // Violation not found.
-        });
```

Comment:
How are we covering the functionality of this deleted test?

(I see in the description that this one is meant to be deleted, but I'm not clear on why.)